src/manager/tooltipManager.js

```diff
@@ -14,28 +14,45 @@ var TooltipManager = class TooltipManager extends MsManager {
     }
     add(actor, params = {}) {
         let actorDestroyed = false;
+        let left = true;
+        let timeoutId = 0;
+        let tooltip;
+        let leaveId;
+        let deactivateId;
```

Comment:
Moving everything at the source of the function it's really not the same in a memory concern. Until now we only at a signal connection for every tool tips and we added more once it's was visible but with your edits we have everything in memory for every tool tips in the shell at any times. It's could be considered as over optimisation but I would prefer to avoid